src/meshcat/geometry.py

```diff
@@ -380,6 +382,43 @@ def data_from_stream(stream):
     return data
 
 
+#we can't make a MeshFileObject that inherits from Object because we need data and resources and no need for material, geom uuids
```

Comment:
Yeah, this suggests that the type hierarchy isn't quite right. Over in the Julia version of meshcat (which is a little more developed), I have `AbstractObject` as a parent type of `Object` and `MeshFileObject`, and we might want something similar here. 

https://github.com/rdeits/MeshCat.jl/blob/6ca2e706d2ccb9501a0fb5ccd5829c208f26af4f/src/mesh_files.jl#L53

https://github.com/rdeits/MeshCat.jl/blob/6ca2e706d2ccb9501a0fb5ccd5829c208f26af4f/src/objects.jl#L6